### ISSUE TITLE
Dashboard tries to fetch protected resources before first login

### ISSUE THREAD

**Issue by petersutter** [kubernetes/dashboard#3680]
##### Environment

```
Installation method: gardener
Kubernetes version: 1.13.4
Dashboard version: 1.10.1
Browser: Chrome 73.0.3683.75 (latest)
```

##### Steps to reproduce

1. Configure `--basic-auth-file` for your kube-apiserver
3. Access kubernetes dashboard via kube-proxy url `https://<domain>/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/`
4. Enter credentials in basic auth popup from browser and confirm 

##### Observed result

After entering the basic auth credentials, you will be prompted two additional times to enter the basic auth credentials
**Chrome:**
![Chrome](https://user-images.githubusercontent.com/5526658/54700113-9c0c2380-4b32-11e9-8c14-f67f16428240.gif)

##### Expected result

You will be prompted only once to enter your credentials in the basic auth popup

##### Comments

When trying to access the kubernetes dashboard for the first time, it tries to fetch resources that can only be accessed successfully **after** you are logged in to the kubernetes dashboard, which results in `401` with www-authorization challenge header in response and `MSG_LOGIN_UNAUTHORIZED_ERROR` error message.
These resources are:
`/api/v1/settings/global`
`/api/v1/rbac/status`

This only happens with Chrome. The stored basic auth credentials are only automatically added to following requests until one of the following requests fails with `401` and then the basic auth credentials are not automatically added to the request. This happened two times in case of the kubernetes dashboard.
Firefox and Safari does not require you to enter the basic auth credentials if one of the requests fail with `401` 
**Firefox:**
![Firefox](https://user-images.githubusercontent.com/5526658/54700042-7b43ce00-4b32-11e9-852a-d121e22fa14f.gif)

**Comment by jcsferraz** [comment]
Hi!

- Are there anothers alternative to resolve that ? I have same problem !

- I think would like to use the --authentication-mode=basic. But how to set up user and password for autentication ?

**Comment by maciaszczykm** [comment]
Closing it as it is connected to the previous version. Could you please verify with v2.0.0-beta1?

**Comment by petersutter** [comment]
@maciaszczykm I can confirm that the issue is solved with v2.0.0-beta1, thanks!

**Comment by maciaszczykm** [comment]
@petersutter Great to hear that.
